test/ConstructiveSolidGeometry/CSG_test.jl

```diff
@@ -177,6 +177,17 @@ using SolidStateDetectors.ConstructiveSolidGeometry:
                     @test !(CartesianPoint{T}(1, 0, -0.1) in prism)
                     @test !(CartesianPoint{T}(1, 0, 1.1) in prism)
                 end
+                @testset "Rectangle" begin
+                    rectangle = RectangleZ(T(1.0), T(2.0), T(3.0)) # x from -0.5..0.5, y from -1.0..1.0, z = 3
```

Comment:
If we use `_is_approx` for `RectangleX` and `RectangleY` and `_eq_z` for `RectangleZ`, I would also suggest testing both here. We might also think about incorporating `CylindricalPoints` in general in the tests.